src/translations/pt.json

```diff
@@ -178,7 +178,7 @@
     "newPassword": "Nova Senha",
     "noFiltersSet": "Nenhum filtro definido",
     "noLabel": "<Nenhum(a) {{label}}>",
-    "noResults": "Nenhum {{label}} encontrado. Ou nenhum(a) {{label}} existe ainda, ou nenhum(a) corresponde aos filtros que você especificou acima.",
+    "noResults": "Nenhum(a) {{label}} encontrado(a). Ou nenhum(a) {{label}} existe ainda, ou nenhum(a) corresponde aos filtros que você especificou acima.",
```

Comment:
The {{label}} for noResults is the plural label of the collection. Does it make sense to add the `(a)` or should it be 'Nenhuma'?

Reply:
Seriously, you're right!
I had done some customization locally and thought it would make sense, but this message doesn't fit.